lib/oper.g

```diff
@@ -1302,6 +1302,78 @@ BIND_GLOBAL( "DeclareAttribute", function ( arg )
     fi;
 end );
 
+##############################################################################
+##
+##  This is a temporary (I hope!) hack for HPCGAP, which introduces thread-local
```

Comment:
HPCGAP vs HPC-GAP

Also, when reading such a comment, it is unclear whom "I" refers to. So perhaps: "This is an hopefully temporary hack"...

Oh, but wait: This is a different version from the comment you added to `hpcgap/lib/oper.g`... So the two are not in sync?! Perhaps I just reviewed the "wrong" version of the comment then?!?

Please sync the two comments, and once you are done, and have pushed an update, I'll go on with the review.